install_ceph.yml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
Could we rename this file to just `ceph_ansible.yml` or `configura_ceph_ansible.yml`?